src/back/msl/writer.rs

```diff
@@ -2275,42 +2315,42 @@ impl<W: Write> Writer<W> {
     ) {
         use crate::Expression;
         self.need_bake_expressions.clear();
-        for expr in func.expressions.iter() {
+        for (fun_handle, expr) in func.expressions.iter() {
```

Comment:
I see that this is what the old `if let` called it, but let's just rename this `expr_handle`, since we no longer know at this point what kind of expression it is.